### DISCUSSION TITLE
Bind global hotkeys (WinUI3)

### DISCUSSION THREAD

**Discussion by steam3d** [HavenDV/H.NotifyIcon#74] [Q&A] [answered]
Is it possible to bind global hotkeys using Win32?

https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-registerhotkey
https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-unregisterhotkey

**Comment by HavenDV** [comment, marked as answer]
I think yes, but you will have to do it yourself as the library doesn't provide support for that. I recommend https://github.com/microsoft/CsWin32 for this.
You can also do a Feature PR if you do so and you want to add support to the library.

If you need an easy way, then you can use my other library https://github.com/HavenDV/H.Hooks

**Reply by steam3d** [reply]
I tested on WInUI3. Global hotkeys work. I am no think that it should be a part of H.NotifyIcon.
